kinto/core/resource/__init__.py

```diff
@@ -1028,7 +1028,7 @@ def _extract_sorting(self, limit):
         modified_field_used = self.model.modified_field in specified
         for field in specified:
             field = field.strip()
-            m = re.match(r"^([\-+]?)([\w\.]+)$", field)
+            m = re.match(r"^([\\-+]?)([\\w\\.]+)$", field)
```

Comment:
Maybe changing from `"` to `'` would be a better fix?

Reply:
Warnings are gone but tests fail :)

Reply:
I don't know what we have in other regexp... single quotes are fine IMO